### PR TITLE
added openshift profile for asm installation

### PR DESCRIPTION
**Pull request by aryan16**

### CODE REVIEW

File: asm/istio/options/openshift.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
```
Comment (bianpengyuan) [inline asm/istio/options/openshift.yaml:1]: s/2020/2022

File: asm/istio/options/openshift.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      namespace: kube-system
+  values:
+    global:
+      pilotCertProvider: "istiod"
+    cni:
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      chained: false
+      cniConfFileName: "istio-cni.conf"
+      excludeNamespaces:
```
Comment (bianpengyuan) [inline asm/istio/options/openshift.yaml:31]: No need to include line 31-33. It is already the default.

File: asm/istio/options/openshift.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - istio-system
+      - kube-system
+      logLevel: info
```
Comment (bianpengyuan) [inline asm/istio/options/openshift.yaml:34]: This is also the default. No need to include this.

File: asm/istio/options/openshift.yaml (hunk 1 above)
Comment (aryan16) [reply asm/istio/options/openshift.yaml:1]: sorry my bad, I directly copied it. Will change it.

File: asm/istio/options/openshift.yaml (hunk 2 above)
Comment (aryan16) [reply asm/istio/options/openshift.yaml:31]: sure

File: asm/istio/options/openshift.yaml (hunk 3 above)
Comment (aryan16) [reply asm/istio/options/openshift.yaml:34]: cool

**Review by bianpengyuan — APPROVED** [review]

**Review by zerobfd — APPROVED** [review]

**Pull request merged**